src/pages/RepoPage/FailedTestsTab/FailedTestsPage/MetricsSection/MetricsSection.tsx

```diff
@@ -1,12 +1,16 @@
-import { useParams } from 'react-router-dom'
+import cs from 'classnames'
```

Comment:
We currently use:

`import { cn } from 'shared/utils/cn'
`
A simple in-place replacement should work